src/mumble/NetworkConfig.cpp

```diff
@@ -79,7 +87,19 @@ void NetworkConfig::load(const Settings &r) {
 }
 
 void NetworkConfig::save() const {
-	s.bTCPCompat         = qcbTcpMode->isChecked();
+	s.bTCPCompat                   = qcbTcpMode->isChecked();
+	s.bUdpVoiceStreamInboundOnly   = qrbUdpIn->isChecked();
+	s.bUdpVoiceStreamOutboundOnly  = qrbUdpOut->isChecked();
+	s.bUdpVoiceStreamBidirectional = qrbUdpBidirectional->isChecked();
+	s.bUdpVoiceStreamAuto          = qrbUdpAuto->isChecked();
+	qgbUdpMode->setDisabled(s.bTCPCompat);
+	fflush(stdout);
```

Comment:
What is this for?

Reply:
My bad again. The fflush(stdout) was not supposed to be there. I am removing it. The rest is just updating the check boxes and radio buttons in the UI.